include/Reader/readerir.h

```diff
@@ -814,6 +814,12 @@ class GenIR : public ReaderBase {
                          bool IsRelocatable, bool IsCallTarget,
                          bool IsFrozenObject = false);
 
+  /// Indicate whether this method needed an unmanaged frame generated for it
+  /// (to make a call to unmanaged code).
+  ///
+  /// \returns True iff an unmanaged frame was created.
+  bool getHasUnmanagedFrame() { return UnmanagedCallFrame != nullptr; }
```

Comment:
Nit: I might call this `containsUnmanagedCall()` for the sake of clarity. Similar feedback applies for the output parameter added above.